content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,318 @@
+===================
+Production analysis
+===================
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |MOs| replace:: :abbr:`MOs (manufacturing orders)`
+
+The *production analysis* report provides statistics about products manufactured using Odoo's
+*Manufacturing* app. The report is useful when trying to understand production costs, manufacturing
+durations, and other important statistics about manufactured products.
+
+To open the production analysis report, navigate to :menuselection:`Manufacturing app --> Reporting
+--> Production Analysis`.
+
+Search... bar
+=============
+
+The :guilabel:`Search...` bar appears at the top of the :guilabel:`Production Analysis` dashboard.
+Along with its drop-down menu, the :guilabel:`Search...` bar used to refine the statistics shown on
+the production analysis report.
+
+By default, the :guilabel:`Search...` bar includes the :guilabel:`End Date: [current year]` filter,
+so that data is only displayed for |MOs| completed within the current calendar year. To remove this
+filter, and show older data as well, click the :icon:`oi-close` :guilabel:`(Remove)` button on the
+right side of the filter.
+
+To only show the data for a specific product, type the product's name in the :guilabel:`Search...`
+bar, and press enter. Multiple products can be entered at the same time, so that only the data for
+those products is shown.
+
+In addition to typing in the :guilabel:`Search...` bar to refine the data shown, clicking the
+:icon:`fa-caret-down` :guilabel:`(toggle)` button on the right side of the bar opens a drop-down
+menu providing additional refinement options. These are organized into the :icon:`fa-filter`
+:guilabel:`Filter`, :icon:`oi-group` :guilabel:`Group By`, and :icon:`fa-adjust`
+:guilabel:`Comparison` sections.
+
+.. image:: production_analysis/search-bar.png
+   :align: center
+   :alt: The Search... bar for the production analysis report.
+
+Filters
+-------
+
+The :guilabel:`Filters` section of the drop-down menu is used to select the time period for which
+production data is displayed. To select a time period, click the :icon:`fa-caret-down`
+:guilabel:`(toggle)` button next to :guilabel:`End Date`, and click one of the options.
+
+The time periods to choose from are the three most recent months, each quarter of the current fiscal
+year (:guilabel:`Q1`, :guilabel:`Q2`, :guilabel:`Q3`, :guilabel:`Q4`), and the three most recent
+years.
+
+Group By
+--------
+
+The :guilabel:`Group By` section of the drop-down menu is used to group manufacturing data into
+specific categories. This makes each graph type show data based on that category, rather than
+individual weeks.
+
+.. example::
+   With the bar chart graph type selected, clicking the :guilabel:`Product` option in the
+   :guilabel:`Group By` section causes the chart to show one bar for each product, instead of one
+   bar for each week during the selected time period.
+
+Comparison
+----------
```

Comment:
Although the pie chart example you showed is quite useful, I personally think comparing between periods is also cool in the pivot table. Perhaps you can include both examples in the doc? 

![image](https://github.com/odoo/documentation/assets/23611711/336b8b06-9812-45f4-9be8-d32ba58cb734)

Reply:
Thanks for pointing this out. I've updated the copy to clarify that the pivot view can be used as well.